### DISCUSSION TITLE
Importing existing templates

### DISCUSSION THREAD

**Discussion by Killerloop2** [TotallyInformation/node-red-contrib-uibuilder#163] [Q&A] [answered]
Hi, I'm looking forward for the required steps to implement an existing template like [(https://coreui.io/)] into my uibuilder notes but nothing seems to work, I've tried the new feature that allows to import external template using Degit to no avail.

Can anyone provide guidance? Has anyone tried to actually import an external template?

Regards

**Comment by TotallyInformation** [comment, marked as answer]
Hi, thanks for getting in touch.

I would advise you to start with my own example repo. This gives you the minimum requirement for an external template. Indeed, I would set up a simple flow and try that template directly first so that you know whether or not there is some other issue at play.

Leaving this open so please do drop in an update as you know more.

Regards, Julian.
